content/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry.md

```diff
@@ -163,6 +163,24 @@ For more information on creating a package, see the [maven.apache.org documentat
 
 {% data reusables.package_registry.viewing-packages %}
 
+### Using a package
+
+To use an Apache Maven package from {% data variables.product.prodname_registry %}, edit the *pom.xml* file to include the package as a dependency. If you want to use packages from more than one repository, add a `repository` tag for each. For more information on using a *pom.xml* file in your project, see "[Introduction to the POM](https://maven.apache.org/guides/introduction/introduction-to-the-pom.html)" in the Apache Maven documentation.
```

Comment:
Thank you so much @skedwards88 ! Right, Maven will try to find the package dependencies in the provided repositories. If it's not found in the first one, it will look for it in the next one. Sorry, I didn't include this information in my pull request :pray: